helix-core/src/test/java/org/apache/helix/store/zk/TestZkHelixPropertyStore.java

```diff
@@ -88,6 +92,57 @@ public void afterClass() {
     deleteCluster(getShortClassName());
   }
 
+  @Test
+  public void testSessionExpirationWithSharedZkClient() throws Exception {
+    /*
+    This test is to make sure SharedZkClient would work with
+    HelixPropertyStore when session expiration happens. More specifically
+    HelixPropertyStore would register all the callbacks after session
+    expiration.
+     */
```

Comment:
This is not how you would enter JavaDoc in a public method. Could you please review the javadoc best practice guidelines?